gdpr.md

```diff
@@ -4,30 +4,30 @@ title: General Data Protection Regulation (GDPR)
 subtitle: GDPR-relevant information for and from joind.in
 ---
 
-## Collected PII
+## <abbr title="General Data Protection Regulation">GDPR</abbr>-relevant information for and from joind.in
```

Comment:
This should have been added to the subtitle above.

Reply:
Cool; PR it :)